relayer/client.go

```diff
@@ -86,7 +86,8 @@ func (c *Chain) CreateClients(dst *Chain) (modified bool, err error) {
 
 	} else {
 		// Ensure client exists in the event of user inputted identifiers
-		_, err := c.QueryClientState(srcH.Header.Height)
+		// TODO: check client is not expired
```

Comment:
I know this will error regardless, but maybe we should create an issue

Reply:
great minds think alike #417. I made one when I added it